csharp/ql/src/change-notes/2025-12-16-path-combine-metadata.md

```diff
@@ -0,0 +1,4 @@
+---
+category: queryMetadata
+---
+* Updated the `name`, `description`, and alert message of PathCombine.ql to have more details about why it's a problem.
```

Comment:
We need to refer to the query by its id `cs/path-combine` instead of the filename (sorry, I didn't notice it the first time around).